app/utils/callback/lava.py

```diff
@@ -685,15 +720,17 @@ def add_tests(job_data, job_meta, lab_name, db_options,
         _get_directory_path(meta, base_path)
         _get_lava_meta(meta, job_data)
         plan_name = meta[models.PLAN_KEY]
+        log = yaml.load(job_data["log"], Loader=yaml.CLoader)
+        results = job_data["results"]
+        _adjust_log_line_numbers(results, log)
         _add_test_log(meta, job_data["log"], plan_name)
         _add_rootfs_info(meta, base_path)
         _store_lava_json(job_data, meta)
         groups = []
         cases = []
         start_log_line = 0
-        log = yaml.load(job_data["log"], Loader=yaml.CLoader)
         end_lines_map = {}
-        for suite_name, results in job_data["results"].iteritems():
+        for suite_name, results in results.iteritems():
```

Comment:
While it works, it can be a bit confusing to reuse the `results` variable name.  Maybe `suite_results` would be clearer?

Reply:
corrected